src/TabsOnEdit.php

```diff
@@ -177,4 +161,31 @@ private function resolveResource(NovaRequest $request)
 
         return $request->newResource();
     }
+
+    private function assignTabPanels(FieldCollection $fields) 
+    {
+        $non_tab_fields = $tab_panels = [];
+
+        $fields->each(function($field, $key) use (&$non_tab_fields, &$tab_panels) {
```

Comment:
Refrain from using this many nested if blocks, return early.

Reply:
@RVxLab I'm not sure I understand this request.

I'm going through the list of ALL fields to see whether or not they need to be left as is (not put into 'tabs' component). But for tabs fields - I wanted to make sure that if there are two or more Tabs() used - that it those fields are assigned properly (there should be as many tabs components as there are Tabs panels)
I can't return here or break - it'll stop assignments for the rest of the block.

Can you please clarify what you meant?